packages/server/src/updater.ts

```diff
@@ -0,0 +1,28 @@
+import * as https from "https";
+
+export const getRecentRelease = (): Promise<string> => {
+	return new Promise<string>((resolve, reject):void  => {
+		https.get({
+			host: "api.github.com",
+			path: "/repos/codercom/code-server/releases/latest",
+			method: "GET",
+			agent: false,
+			headers: {
+				"User-Agent": "code-server",
+			},
+		}, (res) => {
+			if (res.statusCode !== 200) {
+				reject(Error("Failed to acquire release information"));
+			}
+			let body = "";
+			res.on("data", (chunk) => {
+				body += chunk;
+			});
+
+			res.on("end", () => {
+				let release = JSON.parse(body);
```

Comment:
This value is never reassigned. use `const` instead?

```suggestion
				const release = JSON.parse(body);
```